packages/vanilla/tsconfig.cjs.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I based this CJS/ESM setup on the Snyk article: https://snyk.io/blog/best-practices-create-modern-npm-package/?ck_subscriber_id=1697729601